src/doc/reference.md

```diff
@@ -3163,7 +3163,7 @@ An example of a for loop over the contents of an array:
 
 let v: &[Foo] = &[a, b, c];
 
-for e in v.iter() {
+for e in v {
```

Comment:
this technically changes behavior, though I'm not sure it matters here. `&v` would give you `iter()`, just `v` gives `into_iter()`.